src/brsTypes/components/RoSGNode.ts

```diff
@@ -801,6 +802,8 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
         },
     });
 
+    private lookupCI = new Callable("lookupCI", this.lookup.signatures[0]);
```

Comment:
please add docs as well. Also, is this function expected to do the same as `lookup` or what is the difference between the 2?

Reply:
You had one job brightscript, LOL 😆